containertest/containertest.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package containertest provides an ephemeral container (such as a database) for integration testing.
+// It's designed to be used in code that needs to work inside and outside google.
+package containertest
+
+import "io"
+
+// ConnInfo specifies how connect to the created container.
+type ConnInfo struct {
+	Host string
+
+	// PortMapper maps from container port to host port. Do not use after container is closed.
+	PortMapper func(string) string
```

Comment:
Can you make this another type (or name the values)? It's include exactly what the inputs and outputs would be.

Reply:
I'll rename for now. I was thinking there may be value in changing it to an interface in case we want to extend it in the future, but I'm not sure what else would be added, and it will be more characters to use.